src/ApiManagement/ApiManagement.Test/ApiManagement.Test.csproj

```diff
@@ -11,7 +11,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.Management.ApiManagement" Version="6.0.0-preview" />
+    <PackageReference Include="Microsoft.Azure.Management.ApiManagement" Version="7.0.0-preview" />
```

Comment:
There are also references to ApiManagement SDK in ApiManagement.ServiceManagement.csproj and its test project. Please update them all.

Reply:
@KacieKK is working on that project. She will publish a PR soon!

Is this okay?

Reply:
Sorry but I'm afraid then both of your PRs will be blocked.
All dependencies to the same libaray should be updated in one PR, otherwise there will be assembly loading conflicts at runtime, never to mention what if only one of your PRs makes into the release.

Can you update just the reference for her in your PR? Or will it make it unable to compile?